src/app/app-routing.module.ts

```diff
@@ -3,6 +3,9 @@ import {PreloadAllModules, RouterModule, Routes} from '@angular/router';
 
 // These are lazy-loaded routes - note that we dynamic-import the modules here
 // to avoid having an eager dependency on them.
+
+// IMPORTANT: this array is auto-updated by script/generator
```

Comment:
why should the generator change this file? the ten "features" should stay fixed

Reply:
See https://github.com/angular/angular-bazel-example/pull/358#discussion_r273439639